### ISSUE TITLE
In APIResults, ResultData that contains a list of information should be displayed as a json array

### ISSUE THREAD

**Issue by ligadata-william** [LigaData/Kamanja#1149]
Currently, if there are multiple pieces of information returned in an APIResult, such as a list of keys, they are a comma-separated string. For purposes of easy readability and better parsing, this should be changed into an array.

**Comment by ligadata-william** [comment]
@ligadata-dan I don't think this is just your responsibility but you're closest to this at the moment.

**Comment by ligadata-dan** [comment]
@ligadatahari 

Assigning to you..  You may  have already fixed this, if so , just verify

**Comment by ligadatahari** [comment]
https://github.com/LigaData/Kamanja/issues/1165

**Comment by ligadata-william** [comment]
The "Result Data" is still a String instead of an array of keys.
```
{
  "APIResults" : {
    "Status Code" : 0,
    "Function Name" : "GetAllObjectKeys",
    "Result Data" : "Object Keys:com.ligadata.kamanja.samples.models.copdriskassessment.000000000000000001,com.ligadata.kamanja.samples.models.helloworldmodel.000000000000000001,com.ligadata.kamanja.samples.models.copddataingest.000000000000000001",
    "Result Description" : "Successfully fetched all object keys"
  }
}
```

**Comment by ligadata-william** [comment]
Repro Steps:
1. Add Medical Application (bash {KamanjaInstall}/input/SampleApplications/bin/MedicalApp.sh)
2. Add Hello World Application (bash {KamanjaInstall}/input/SampleApplications/bin/HelloWorldApp.sh)
3. Run get all models (bash {KamanjaInstall}/bin/kamanja get all models)
